### DISCUSSION TITLE
Where do I put the favicon?

### DISCUSSION THREAD

**Discussion by mateoconlechuga** [CaiJimmy/hugo-theme-stack#643] [Q&A] [answered]
Hello, I was wondering where I should put the `favicon.ico` for my site? I imagine the `static` folder - but I'm not sure if I also need to edit the `config.yaml` as well. Thank you for an awesome theme!

**Comment by Sp-cy** [comment, marked as answer]
I have it inside a folder called img and then referenced it in the config like this:

> favicon: /img/favicon.ico

**Reply by mateoconlechuga** [reply]
Thanks!
